doc/nrf/app_dev/device_guides/nrf54l/pinmap.rst

```diff
@@ -0,0 +1,50 @@
+.. _ug_nrf54l_pinmap:
+
+nRF54L pin mapping
```

Comment:
If it has to go in before rc2, this addition needs to be mentioned in the release notes PR https://github.com/nrfconnect/sdk-nrf/pull/25910